### ISSUE TITLE
please avoid unnecessary complications

### ISSUE THREAD

**Issue by h1uke** [Ansersion/myRtspClient#22]
Current build configuration file for linux suggests that the C++11 compiler is required in order to build:

> config.linux: COMPILE_OPTS =	$(INCLUDES) -I. -O2 -std=c++11 -fPIC

but, in reality, only one single line of code cannot be compiled by an older gcc v4.8 that we're only allowed to use in our research, and which doesn't understand C++11.

> rtspClient.cpp:110   RtspClient::RtspClient(string uri): RtspClient()

The above is a minor problem which can be fixed in no time.

Author: if you do not mind, please remove the C++11 dialect requirement. Thanks.

**Comment by Ansersion** [comment]
You are right. It's a little inconvenience for some compiler. I've modified it just now.
